protoc.go

```diff
@@ -35,7 +35,7 @@ var (
 		{{- if $index}}+{{end}}
 		{{- $plugin}}
 	{{- end -}}
-	,{{- end -}}import_path={{.ImportPath}}
+	,{{- end -}}
```

Comment:
I'm not really sure the effect of this removal. Apparently protobuild works without the flag.